docs/source/index.rst

```diff
@@ -16,9 +16,30 @@ Welcome to ablator's documentation!
    ablator.modules
    ablator.utils
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Tutorials
+
+   notebooks/GettingStarted.ipynb
+   notebooks/Environment-settings.ipynb
+   notebooks/Configuration-Basics.ipynb
+   notebooks/Prototyping-models.ipynb
+   notebooks/Search-space-tutorial.ipynb
+   notebooks/HPO-tutorial.ipynb
+   notebooks/Experiment-dir.ipynb
+   notebooks/Interpreting-results.ipynb
+   notebooks/GettingStarted-more-demos.ipynb
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Intermediate Tutorials
+
+   notebooks/Searchspace-for-diff-optimizers.ipynb
+
 Indices and tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+ :ref:`search`
```

Comment:
was this added by mistake? It seems to be repeating?

Reply:
It's a mistake, fixed in the new commit